controllers/sr_controller/sr_controller.py

```diff
@@ -115,10 +116,24 @@ def print_simulation_version() -> None:
     print("Running simulator version {}".format(version))
 
 
+def reconfigure_environment(robot_file: Path) -> None:
+    """
+    Reconfigure the interpreter environment for the actual location of the
+    competitor code.
+    """
+
+    # Remove ourselves from the path and insert the competitor code
+    sys.path.pop()
```

Comment:
Is it standard for our directory to always be the first? I wonder whether a `.remove` would be safer here?

Alternatively, do we need to remove ourselves?

Reply:
> Is it standard for our directory to always be the first?

Yeah, I believe so. If it wasn't well defined you could end up with some very weird/subtle import bugs on different platforms.

Reply:
So, it seems that we both missed the actual bug here. This line is the cause of #167 because `pop` changes the _last_ item in the list, not the first.